### PR TITLE
docs: add a Common Pitfalls section to README

### LINKED ISSUE #134: documentation should say "do not put spaces in your home directory"
```
Hook 'vpython.bat src/electron/script/apply_all_patches.py src/electron/patches/config.json' took 15669.14 secs
subprocess2(154) __init__:python3 src/electron/script/update-external-binaries.py;  cwd=C:\src\electron
________ running 'python3 src/electron/script/update-external-binaries.py' in 'C:\src\electron'
Downloading https://electron-build-tools.s3-us-west-2.amazonaws.com/build-dependencies/b6a20fd1c2026f3792e7286bc768a7ebc261847b76449b49f55455e1f841fecd/sccache-win32-x64.zip to C:\Users\NICOLE~1\AppData\Local\Temp\electron-xzab9lsb\sccache-win32-x64.zip
Download sccache-win32-x64.zip     4151947  [103798675.0%]
Hook 'python3 src/electron/script/update-external-binaries.py' took 11.25 secs
subprocess2(154) __init__:vpython.bat -c import os, subprocess; os.chdir(os.path.join("src", "electron")); subprocess.check_call(["python", "script/lib/npx.py", "yarn@1.15.2", "install", "--frozen-lockfile"]);;  cwd=C:\src\electron
________ running 'vpython.bat -c import os, subprocess; os.chdir(os.path.join("src", "electron")); subprocess.check_call(["python", "script/lib/npx.py", "yarn@1.15.2", "install", "--frozen-lockfile"]);' in 'C:\src\electron'
Error: EPERM: operation not permitted, mkdir 'C:\Users\Nicole'
```

### PR DESCRIPTION
**Pull request by ckerr**
... and say "do not put spaces in your home directory"

This was the consensus answer for 134 in Vancouver

Fixes #134

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -434,3 +434,7 @@ Regardless of whether automatic updates are enabled, you can manually call the f
 ```
 e auto-update check
 ```
+
+# Common Pitfalls
```
Comment (dsanders11) [inline README.md:438]:
```suggestion
## Common Pitfalls
```

**Review by dsanders11 — APPROVED** [review]

**Review by MarshallOfSound — COMMENTED** [review]
Can we codify this too? Just in `e init` maybe, checking `process.env.HOMEDIR` for a space or whatever the x-platform way of doing that is

**Comment by codebytere** [comment]
@ckerr do you still want to move this forward?

**Comment by dsanders11** [comment]
#444 could be combined with this, IMO.

**Pull request closed without merging**